packages/docs/celo-holder-guide/cusd.md

```diff
@@ -26,23 +26,23 @@ After creating an account, record its address in environment variables:
 export CELO_ACCOUNT_ADDRESS=<YOUR-CELO-ACCOUNT-ADDRESS>
 ```
 
-## Exchange CELO for cUSD
+## Exchange CELO for Stablecoins
 
 Once you have deposited CELO to your account, you can check your balance:
 
 ```
 celocli account:balance $CELO_ACCOUNT_ADDRESSS
 ```
 
-You can exchange CELO for cUSD using the following command. Note that the unit of value is CELO Wei (1 CELO = 10^18 CELO Wei).
+As an example of a common stablecoin swap, you can exchange CELO for cUSD using the following command. This exchanges CELO for stable tokens (cUSD by default) via the stability mechanism. Note that the unit of value is CELO Wei (1 CELO = 10^18 CELO Wei).
 
 ```
 celocli exchange:celo --value <VALUE-TO-EXCHANGE> --from $CELO_ACCOUNT_ADDRESS
 ```
 
-## Transfer cUSD
+## Transfer Native Assets
```

Comment:
Are the terms "stable coin" and "native assets" interchangeable? Suggest sticking with one term or somewhere explaining the difference/similarities between the two

Reply:
Changed back to "stablecoins" for congruency.